src/test/java/trello/steps/CardSteps.java

```diff
@@ -63,7 +63,8 @@ public void addCardToList(final String listTitle, final String cardTitle) {
      */
     @Then("I should see the new card with the given task")
     public void verifyNewCardisDisplayinList() {
```

Comment:
Please, use camel case in name of method.

Reply:
done